rubygem-net-ssh-krb/rubygem-net-ssh-krb.spec

```diff
@@ -0,0 +1,93 @@
+# Generated from net-ssh-krb-0.4.0.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name net-ssh-krb
+%global smart_proxy_dynflow_bundlerd_dir %{?rhel:/opt/theforeman/tfm/root/}%{_datadir}/smart_proxy_dynflow_core/bundler.d
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.4.0
+Release: 1%{?dist}
+Summary: Add Kerberos support to Net::SSH
+Group: Development/Languages
+License: GPLv2
+URL: http://github.com/cbeer/net-ssh-kerberos
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(net-ssh) >= 2.0
+Requires: %{?scl_prefix}rubygem(gssapi) >= 1.2.0
+Requires: %{?scl_prefix}rubygem(gssapi) < 1.3
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%description
+Extends Net::SSH by adding Kerberos authentication capability for
+password-less logins on multiple platforms.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+mkdir -p %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}
+cat <<EOF > %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}/%{gem_name}.rb
+gem '%{gem_name}'
+EOF
+
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%license %{gem_instdir}/LICENSE
+%{gem_instdir}/example
+%{gem_libdir}
+%exclude %{gem_instdir}/net-ssh-kerberos.gemspec
+%exclude %{gem_cache}
+%{gem_spec}
+%{smart_proxy_dynflow_bundlerd_dir}/%{gem_name}.rb
```

Comment:
This part seems a bit strange tieing this so closely to smart_proxy_dynflow. This seems like something that maybe it should be doing and let this be a stand alone gem?